packages/timeline-state-resolver/src/integrations/tricaster/externalStateConverter.ts

```diff
@@ -0,0 +1,160 @@
+import {
+	TriCasterAudioChannelName,
+	TriCasterDelegateName,
+	TriCasterInputName,
+	TriCasterKeyerName,
+	TriCasterLayerName,
+	TriCasterMixEffectName,
+	TriCasterMixOutputName,
+} from 'timeline-state-resolver-types'
+import { ElementCompact, xml2js } from 'xml-js'
+import {
+	fillRecord,
+	TriCasterMixEffectState,
+	TriCasterAudioChannelState,
+	TriCasterInputState,
+	TriCasterKeyerState,
+	TriCasterLayerState,
+	TriCasterMixOutputState,
+	TriCasterState,
+} from './triCasterStateDiffer'
+import { CommandName, TriCasterGenericCommandName } from './triCasterCommands'
+
+type ShortcutStates = {
+	[key: string]: string | number
+}
+
+export class ExternalStateConverter {
+	private shortcutStates: ShortcutStates
```

Comment:
Do you need this instance variable? By having it you are saving state in this classe (which may be needed/justified), but stateless classes is much prefered over statefull.

You only have one public method in this class and that method sets this variable, which indicates to me it may not be needed :)